kernel/arch/i386/boot/bootparse.cpp

```diff
@@ -270,11 +270,11 @@ void px_parse_stivale2(void *info)
             case STIVALE2_STRUCT_TAG_PXE_SERVER_INFO:
             {
                 auto pxe = (struct stivale2_struct_tag_pxe_server_info*)tag;
-                unsigned char ip[4];
-                ip[0] = pxe->server_ip & 0xFF;
-                ip[1] = (pxe->server_ip >> 8) & 0xFF;
-                ip[2] = (pxe->server_ip >> 16) & 0xFF;
-                ip[3] = (pxe->server_ip >> 24) & 0xFF;
+                uint8_t ip[4];
+                ip[0] = (uint8_t)(pxe->server_ip & 0xFF);
+                ip[1] = (uint8_t)((pxe->server_ip >> 8) & 0xFF);
+                ip[3] = (uint8_t)((pxe->server_ip >> 24) & 0xFF);
```

Comment:
Why reverse the order of these two lines?

Reply:
Was accidental. Probably accidentally swapped when trying to do a keyboard macro.

Reply:
Resolved in fe71257.